backend/tests/services/test_local_device_command_service.py

```
@@ -564,6 +574,122 @@ def test_ls_skills_command_parses_yaml_block_description(tmp_path):
+        """---
+name: shared-context
+description: Shared local context.
+---
+
+# Shared Context
+""",
+        encoding="utf-8",
+    )
+
+    env = {**os.environ, "HOME": str(tmp_path)}
+    result = subprocess.run(
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)